lib/matplotlib/testing/widgets.py

```diff
@@ -83,3 +83,31 @@ def do_event(tool, etype, button=1, xdata=0, ydata=0, key=None, step=1):
     event = mock_event(tool.ax, button, xdata, ydata, key, step)
     func = getattr(tool, etype)
     func(event)
+
+
+def click_and_move(tool, start, end, key=None):
```

Comment:
```suggestion
def click_and_drag(tool, start, end, key=None):
```
I think moving the mouse while holding a button is called "drag".